source/_patterns/01-molecules/navigation/view-selector.mustache

```diff
@@ -1,29 +1,40 @@
-<div class="view-selector{{#secondaryUrl}} view-selector--has-secondary{{/secondaryUrl}}" data-behaviour="ViewSelector"{{#sideBySideUrl}} data-side-by-side-link="{{sideBySideUrl}}"{{/sideBySideUrl}}>
+<div class="view-selector{{#secondaryUrl}} view-selector--has-secondary{{/secondaryUrl}}{{#isTabSelector}} button--switch-selector{{/isTabSelector}}" data-behaviour="ViewSelector"{{#sideBySideUrl}} data-side-by-side-link="{{sideBySideUrl}}"{{/sideBySideUrl}}>
   <ul class="view-selector__list">
-    <li class="view-selector__list-item view-selector__list-item--primary{{^secondaryIsActive}} view-selector__list-item--active{{/secondaryIsActive}}">
-      <a href="{{primaryUrl}}" class="view-selector__link view-selector__link--primary">{{primaryLabel}}</a>
-    </li>
-    {{#secondaryUrl}}
+    {{#isTabSelector}}
+      <li class="view-selector__list-item view-selector__list-item--primary{{^secondaryIsActive}} view-selector__list-item--active{{/secondaryIsActive}}">
+        <span class="view-selector__link">{{primaryLabel}}</span>
+      </li>
       <li class="view-selector__list-item view-selector__list-item--secondary{{#secondaryIsActive}} view-selector__list-item--active{{/secondaryIsActive}}">
-          <a href="{{secondaryUrl}}" class="view-selector__link view-selector__link--secondary">{{secondaryLabel}}</a>
+        <span class="view-selector__link">{{secondaryLabel}}</span>
       </li>
-    {{/secondaryUrl}}
+    {{/isTabSelector}}
 
-    {{#otherLinks}}
-      <li class="view-selector__list-item">
-        <a href="{{url}}" class="view-selector__link"><span>{{{name}}}</span></a>
+    {{^isTabSelector}}
```

Comment:
Everything inside {{^isTabSelector}} wapper is not changed, the old logic is moved there without updates, the only new logic is inside {{#isTabSelector}}